README.adoc

```diff
@@ -219,6 +219,31 @@ treasury (5 BPS; governable), and updates the user balances with information
 from `DepositSweep.processDepositSweepTxInputs` and
 `DepositSweep.depositSweepTxFeeDistribution`.
 
+==== Redeeming
+
+An account with a balance can request a redemption via
+`Bridge.requestRedemption`. We verify that the destination is valid (P2PKH,
+P2WPKH, P2SH or P2WSH), and build a redemption key based on the wallet's PKH
+and destination. There can only be one pending redemption per PKH-destination
+pair. The treasury takes a cut (`Bridge.redemptionTreasuryFeeDivisor`, 5 BPS),
+and then we reduce the account's balance and start a timer.
+
+After `Bridge.redemptionTimeout` (48 hours), anyone may call
+`Bridge.notifyRedemptionTimeout`. This will decrease
+`wallet.pendingRedemptionsValue`, mark the redemption as "timed out", punish
+the operators for `Bridge.redemptionTimeoutSlashingAmount`, and reward the
+notifier for a percentage (`Bridge.redemptionTimeoutNotifierRewardMultiplier`,
+100%) of the slashed stake. The redeemer is reimbursed the balance of the
+redemption, and the wallet begins to move its funds and eventually close via
+`Wallets.notifyWalletTimedOutRedemption`.
```

Comment:
Err... not sure what we mean here by being eventually closed. The wallet that moved funds gets closed when they finalize the closing period (MOVING_FUNDS -> CLOSING -> CLOSED)

Reply:
I meant that this this kicks the process off where closing is the sort of end destination, but I see how that's confusing. I'll change the wording!